app/components/SearchDropdown.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import {Typeahead} from 'react-bootstrap-typeahead';
+
+interface Props {
+  placeholder?: string;
+  options: string[];
```

Comment:
This attribute [should accept an array of objects](https://github.com/ericgio/react-bootstrap-typeahead/blob/master/docs/Data.md) as well.

Reply:
Updated.